cmd/deplist/deplist.go

```diff
@@ -36,13 +36,21 @@ func main() {
 			version := dep.Version
 
 			inst, _ := purl.FromString(fmt.Sprintf("pkg:%s/%s@%s", deplist.GetLanguageStr(dep.DepType), dep.Path, version))
-			fmt.Println(inst)
```

Comment:
Perhaps instead of changing the output format for embedded dependencies we should add a command flag which indicates if we want embedded dependencies in the output of not? I think it would avoid the need to have special code in clients which for interrupting the output.

Reply:
@jasinner I could add a flag `-modes` which accepts `deps`, `bundled`, or `deps,bundled` and prints dependencies based on the mode. However, for the case where you print both "regular dependencies" and bundled code you do need anyway a way to differentiate them, don't you? Opinions @jasinner ? @sfowl was suggesting to have a JSON output instead.

Do we have clients that use the output mode of deplist?

Reply:
We are planning on using deplist in [component-registry](https://github.com/RedHatProductSecurity/component-registry). Although the integration is not done yet.

I think for component-registry it would be nice to have json output and split deps and bundled into 2 groups.